src/en/users-auth.md

```diff
@@ -24,22 +25,90 @@ juju bootstrap aws mycontroller
 juju change-user-password
 ```
 
-See [Creating a controller](./controllers-creating.html) for details on
-controller creation.
+See [Creating a controller][controllers-creating] for details on controller
+creation.
 
+## Credentials
 
-## Credentials and SSH keys
-
-*Credentials* managed in Juju are related to the accessing of the chosen
+Credentials managed in Juju are related to the accessing of the chosen
 cloud-backed resource. They are not related to Juju users themselves in any
 way. Credentials added to Juju remain local to the system user (on Ubuntu:
 `~/.local/share/juju/credentials.yaml`).
 
-See [Cloud credentials](./credentials.html) for more on this topic.
+See [Cloud credentials][credentials] for more on this topic.
+
+## SSH access
+
+SSH access is managed on a per-model basis. That is, if a public key is added
+to a model then that key is placed on all machines (present and future) in that
+model.
+
+Each Juju machine provides a user account named 'ubuntu' and it is to this
+account that public keys are added when using the Juju SSH commands (
+`juju add-ssh-key` and `juju import-ssh-key`). Because this user is effectively
+the 'root' user (passwordless sudo privileges), the granting of SSH access must
+be done with due consideration.
+
+### Native ssh
```

Comment:
I don't think 'Native' quite describes it properly. Maybe 'Direct SSH access'? or something

Reply:
OK